recipe/meta.yaml

```
@@ -106,9 +106,11 @@ outputs:
       run:
         - python
         - {{ pin_subpackage(name, max_pin='x.x.x.x.x.x') }}
+        # Try match what's in https://github.com/apache/beam/blob/master/sdks/python/setup.py#L261
```

```suggestion
        # Try match what's in https://github.com/apache/beam/blob/v{{ version }}/sdks/python/setup.py#L268
```
It's important to match the specific version number, rather than the `master` branch.  But you're absolutely right that this got missed.

---

@xylar awesome - thanks for catching that :)